RepoDb.Core/RepoDb/ClassMap.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Linq.Expressions;
+
+namespace RepoDb
+{ 
+    public class ClassMap<T> where T : class
```

Comment:
Hey @guy-lud , would you be able to provide an XML comments? Though I can provide it as well after the PR, just do let me know.

Reply:
@mikependon, yes i can :) but i am the worst "writer" ever. Would you like me to add it so you can edit them?

Reply:
No worried then, I can manage. Thanks :)